### PR TITLE
Update your ALSOs

### PR DESCRIPTION
**Pull request by ooffordable**
by oooffordable

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,8 @@
 ### Welcome to Jae-Won's profile 🎉
 
-An undergraduate interested in **software systems and deep learning**. Also a fingerstyle guitar player.
+An undergraduate interested in **software systems and deep learning**. Also a fingerstyle guitar player. 
+
+
+Also a main TA of [OSSPR2021](https://github.com/swsnu/osspr2021).
```
Comment (jaywonchung) [inline README.md]: a -> the please

File: README.md [hunk 2]
```diff
@@ -1,3 +1,8 @@
...
+Also a member of the blood-forged ally of [CRANE](https://github.com/snuspl/crane)
```
Comment (jaywonchung) [inline README.md]: Missing period please

File: README.md [hunk 3]
```diff
@@ -1,3 +1,8 @@
...
+Also a nvim spreader in SPLab.
```
Comment (jaywonchung) [inline README.md]:
```markdown
Also a spreader of the Neovim plague in [SPLab](https://spl.snu.ac.kr).
```
please

**Review by jaywonchung — CHANGES_REQUESTED** [review]
Thank you for your contribution. I left some change requests. Take a look!

**Comment by ooffordable** [comment]
> Thank you for your contribution. I left some change requests. Take a look!

Thanks for your timely review. I addressed all of your comments.
Would you please merge my modification to your master branch?

**Review by jaywonchung — APPROVED** [review]
LGTM! I believe that these ALSO statements and you being the contributor summarize my time at SPL very well. I sincerely appreciate your contribution.

**Pull request merged**